dir/path.go

```diff
@@ -77,7 +76,24 @@ const (
 	TrustStoreDir = "truststore"
 )
 
-var userConfigDir = os.UserConfigDir // for unit test
+// The relative path to {NOTATION_LIBEXEC}
+const (
+	// PathPlugins is the plugins directory relative path.
+	PathPlugins = "plugins"
+)
+
+// The relative path to {NOTATION_CACHE}
+const (
+	// PathCRLFileCache is the crl file cache directory relative path.
+	PathCRLFileCache = "crl"
```

Comment:
Since it is always "file", we may name it
```suggestion
	PathCRLCache = "crl"
```

Reply:
Updated.